test/test_container.py

```diff
@@ -118,6 +120,11 @@ def logger():
         yield patched
 
 
+@pytest.fixture(autouse=True)
```

Comment:
The "default" behaviour which django enables there also only shows the warning once, so subsequent trips through the same code path don't trigger it. To fix the currently failing tests I need to set it to "always" mode.

I have added it as a top-level fixture (rather than in `pytest_configure`) so that it can be disabled by overriding if needs be.